dash_bio/component_factory/_clustergram.py

```diff
@@ -384,7 +384,7 @@ def figure(self, computed_traces=None):
         # [row dendro] [heatmap]     [heatmap]     [row GM]
         # [row dendro] [heatmap]     [heatmap]     [row GM]
         # [empty]      [col. GM]     [col. GM]     [empty]
-        fig = tools.make_subplots(
+        fig = subplots.make_subplots(
```

Comment:
Oh nice, presumably this will remove the warning you get when running dash-docs locally

```
/Users/alex/Library/Python/3.8/lib/python/site-packages/plotly/tools.py:461: DeprecationWarning:

plotly.tools.make_subplots is deprecated, please use plotly.subplots.make_subplots instead
```

Though - you see this warning twice in a row, maybe there's another use of `tools` in `dash-bio`?

Reply:
Nah, looks like only one `make_subplots` anywhere in `dash-bio` - so the docs probably just call this twice.